tools/user-scripts/src/index.ts

```diff
@@ -0,0 +1,24 @@
+import { Effect } from 'effect';
+import { UserRuntime, UserService } from './lib/user-scripts.js';
+
+export const deleteUser = (userId: string) =>
+  UserRuntime.runPromise(
+    UserService.pipe(
+      Effect.flatMap((userService) =>
+        Effect.retry(userService.deleteUser(userId), {
+          times: 5,
+        }),
+      ),
+    ),
+  );
+
+export const getUsers = (filterTerm: string, query: string) =>
+  UserRuntime.runPromise(
+    UserService.pipe(
+      Effect.flatMap((userService) =>
+        Effect.retry(userService.getUsers(filterTerm, query), {
```

Comment:
What are `filterTerm` and `query` here? Can you document these in the README and give examples?